doc/source/whatsnew/v1.3.0.rst

```diff
@@ -695,7 +695,7 @@ Conversion
 - Bug in :class:`Index` construction silently ignoring a passed ``dtype`` when the data cannot be cast to that dtype (:issue:`21311`)
 - Bug in :meth:`StringArray.astype` falling back to numpy and raising when converting to ``dtype='categorical'`` (:issue:`40450`)
 - Bug in :class:`DataFrame` construction with a dictionary containing an arraylike with ``ExtensionDtype`` and ``copy=True`` failing to make a copy (:issue:`38939`)
--
+- Bug in :func:`_coerce_to_type` failing to convert ``Float64DType`` input into numpy array (:issue:`40730`)
```

Comment:
Whatsnew entries should reference behavior that users see. So instead of an internal function, it should mention the effect on `qcut`, something like `Bug in :meth:`pandas.qcut` raising for nullable float types`

Reply:
Got it. Will resolve it accordingly.